guides/create-a-tax-plugin.md

```diff
@@ -0,0 +1,138 @@
+## At a Glance
+Most companies need to collect and report sales tax, and because of the complexities of understanding
```

Comment:
one extra line break

Reply:
Unclear on where the extra line break is

Reply:
Tried to adjust this based on what I thought you might have meant. Take a look if that's right